src/app/zap-templates/zcl/data-model/chip/zone-management-cluster.xml

```diff
@@ -94,16 +95,19 @@ Git: 0.9-fall2024-302-gebc8548b3
     <description>This cluster provides an interface to manage regions of interest, or Zones, which can be either manufacturer or user defined.</description>
     <client init="false" tick="false">true</client>
     <features>
-      <feature bit="0" code="TWODCART" name="TwoDimensionalCartesianZone" summary="Devices support Two Dimensional Cartesian Zones">
-        <mandatoryConform/>
+      <feature bit="0" code="TWODCART" name="TwoDimensionalCartesianZone" summary="Support Two Dimensional Cartesian Zones">
+        <optionalConform choice="a" more="true" min="1"/>
```

Comment:
I know that's what the spec says, but it's nonsense.  You can only use choice conformance when more than one thing is labeled with the same choice, no?